.github/ISSUE_TEMPLATE/7_cherry_pick.yml

```diff
@@ -0,0 +1,72 @@
+name: Request a cherry-pick.
+description: As a contributor, you would like to request that a feature be cherry-picked into a release.
+title: '[CP] <title>'
+labels: ['cp: review']
+body:
+  - type: input
+    id: issue_link
+    attributes:
+      label: issue_link
+      description: What is the link to the issue this cherry-pick is addressing?
+      placeholder: https://github.com/flutter/flutter/issues/12345
+    validations:
+      required: true
+  - type: input
+    id: commit_hash
+    attributes:
+      label: Commit Hash
+      description: What is the shortened commit hash that has been merged to master/main?
+      placeholder: fabfeed
+    validations:
+      required: true
+  - type: input
+    id: impacted_users
+    attributes:
+      label: Impacted Users
+      description: Approximately who will hit this issue (ex. all Flutter devs, Windows developers, all end-customers, apps using X framework feature)?
+      placeholder: All Flutter devs
+    validations:
+      required: true
+  - type: textarea
+    id: impact_description
+    attributes:
+      label: Impact Description
+      description: What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. flutter doctor crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)
+      placeholder: Android apps can not compile without this fix.
+    validations:
+      required: true
+  - type: textarea
+    id: workaround
+    attributes:
+      label: Workaround
+      description: Is there a workaround for this issue?
+      placeholder: Reload the app
+    validations:
+      required: true
+  - type: dropdown
+    id: risk
+    attributes:
+      label: Risk
+      description: What is the risk level of this cherry-pick?
+      options:
+        - low
+        - medium
+        - high
+    validations:
+      required: true
+  - type: dropdown
+    id: test_coverage
+    attributes:
+      label: Test Coverage
+      description: Are you confident that your fix is well-tested by automated tests?
+      options:
+        - "yes"
```

Comment:
Maybe a link to the tests in the PR? developers may be tempted to say yes even without checking

Reply:
The tests will show up in the PR, I don't personally think adding a link will deter this behavior.